### ISSUE TITLE
doc: Consider removing dangerous Buffer usage from the Readme

### ISSUE THREAD

**Issue by ChALkeR** [form-data/form-data#330]
I'm talking about the examples on https://github.com/form-data/form-data#usage.

Sending `new Buffer(10)` to a remote host is unsafe, it basically sends a chunk of process memory, which potentially cointains db passwords, encryption keys, source file chunks, etc.

While not a vulnerability in `form-data`, having that code as the first example of how to use this library is probably not a very good idea.

**Comment by ChALkeR** [comment]
I'm not exactly sure, but this could be caused by the examples here:
 * `httpsync-0.0.8.tgz/example/upload_file.js:25:form.append('my_buffer', new Buffer(10));`
 * https://github.com/Montage-Inc/javascript-montage/blob/master/tests/client_test.js#L337

**Comment by alexindigo** [comment]
Thank you for your concern. Looks like it's either another example or a test that send data to itself.
And that example, is just to show in the simple terms how FormData works with Buffers. No "production" code would use it, since it doesn't make sense in the real world and provides no value.
But if you think there is a simple way to illustrate how it works with Buffers without it being a security issue, please let us know. Thank you.

**Comment by seishun** [comment]
>But if you think there is a simple way to illustrate how it works with Buffers without it being a security issue, please let us know.

`Buffer.alloc(10)`

**Comment by motet-a** [comment]
In addition to be unsafe, [it is deprecated](https://nodejs.org/api/buffer.html#buffer_new_buffer_size).

**Comment by DylanPiercey** [comment]
@motet-a I agree, we should probably remove this deprecated api. I plan on doing some cleanup on this lib when I get a bit of time but feel free to make a PR.
